### PR TITLE
Document developer configuration parameters

### PR DESCRIPTION
**Pull request by christiam**
Document additional documentation parameters

### CODE REVIEW

File: docs/source/configuration.rst [hunk 1]
```diff
@@ -650,6 +650,48 @@ Developer configuration options
...
+
+``ELB_JANITOR_SCHEDULE``
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+    **Set via an environment variable**.
+
+    * Default: run every 5 minutes.
+
+      * For GCP: ``*/5 * * * *``
+      * For AWS: ``cron(*/5 * * * ? *)``
```
Comment (victzh) [inline docs/source/configuration.rst:675]: Would not it be better to unify this? Make this parameter a source for cron(x) for AWS.

File: docs/source/configuration.rst [hunk 2]
```diff
@@ -650,6 +650,48 @@ Developer configuration options
...
+    **Set via an environment variable**.
+
+    * Default: Disabled
+    * Values: Any string. Set to any value to enable.
+
+    Disables the auto-shutdown feature.
+
+.. _elb_janitor_schedule:
+
+``ELB_JANITOR_SCHEDULE``
```
Comment (boratyng) [inline docs/source/configuration.rst:667]: Should we expose this one to users?

File: docs/source/configuration.rst (hunk 1 above)
Comment (christiam) [reply docs/source/configuration.rst:675]: @victzh  - Can you please elaborate on to unify this?

File: docs/source/configuration.rst (hunk 2 above)
Comment (christiam) [reply docs/source/configuration.rst:667]: This is not really an end-user configuration parameter, hence its location in the developer configuration options section.

File: docs/source/configuration.rst (hunk 1 above)
Comment (victzh) [reply docs/source/configuration.rst:675]: To have */x * * * * for both GCP and AWS, but in AWS wrap user set value in cron(x). Do you see any usefulness in setting AWS parameter to anything else than cron?

Comment (christiam) [reply docs/source/configuration.rst]: Thanks for the clarification, yes, I agree that that would be useful and potentially safer. Leaving it as is leave us more flexibility for different scheduling configurations (e.g.: one off, daily, etc).

**Pull request closed without merging**